src/ui.rs

```diff
@@ -68,7 +69,7 @@ pub fn image_uv_tilesize(texture: Texture, graphic: Graphic, text_size: f32) ->
                 .unwrap_or((0, 0));
             (x, y, tilemap_width, tilemap_height, tilesize)
         }
-        texture => {
+        texture @ Texture::Egui => {
```

Comment:
What's the reason for the extra `@ Texture::Egui` here?

Reply:
Clippy calculate that as a single variant that could represent future things but i will remove it.